packages/react-keytips/src/utilities/isTargetVisible.ts

```diff
@@ -6,7 +6,8 @@ export const isTargetVisible = (
 ): boolean => {
   if (!target || !win) return false;
   if (isDisabled(target)) return false;
+  if (target?.parentElement?.hasAttribute('[data-overflowing]')) return false;
```

Comment:
[checkVisibility](https://developer.mozilla.org/en-US/docs/Web/API/Element/checkVisibility) could be a better option, if not our browser matrix support and jsDOM not supporting it [yet](https://github.com/jsdom/jsdom/issues/3695) 😞